### PR TITLE
PE-2749: fix for CVEs

### PR DESCRIPTION
**Pull request by LochanRn**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,6 +5,7 @@ RUN \
   apk add --virtual=build libffi-dev make && \
   apk add curl jq unzip wget && \
   apk del --purge build
+RUN apk add --no-cache cdrkit p7zip
```
Comment (abhinavnagaraj) [inline Dockerfile:8]: where is this used ?

Comment (LochanRn) [reply Dockerfile:8]: cdrkit is used by libvirt to create disks, mkisof is a tool i guess.

Comment (abhinavnagaraj) [reply Dockerfile:8]: I meant why is this required for CVE fix ?

Comment (LochanRn) [reply Dockerfile:8]: It was there before also, i removed it unknowing to the fact that it is required. 

If you observe the docker file there are a lot of un required pkgs that were being installed eg: python, sqllite etc. 
Removed all of them as they are not required.

**Review by abhinavnagaraj — APPROVED** [review]

**Pull request merged**